axelrod/history.py

```diff
@@ -0,0 +1,88 @@
+from collections import Counter
+
+from axelrod.action import Action, actions_to_str
+
+C, D = Action.C, Action.D
+
+
+class History(object):
+    """
+    History class to track the history of play and metadata including
+    the number of cooperations and defections, and if available, the
+    opponents plays and the state distribution of the history of play.
+    """
+
+    def __init__(self, plays=None, coplays=None):
```

Comment:
Could we document this, (it wasn't immediately obvious to me what a `coplay` was).

Reply:
Sure